src/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.tsx

```diff
@@ -115,12 +150,22 @@ const AddProfessionalCategoryModal: FC<AddProfessionalCategoryModalProps> = ({
     })
   }
 
+  const handleSubjectDelete = (id: string) => {
+    handleDataChange({
+      subjects: data.subjects.filter((el) => el._id !== id)
```

Comment:
Maybe would be beneficial to move the logic to a separate variable for readability and debugging purposes like this: 

```
const updatedSubjects = data.subjects.filter((el) => el._id !== id);

  handleDataChange({
    subjects: updatedSubjects
  });
```

Reply:
Thank you fixed!